src/components/HistoryTables/MsigProposals/Proposal.tsx

```diff
@@ -80,24 +80,18 @@ export default function ProposalDetail(props: ProposalDetailProps) {
     return _msigTxsError
   }, [_msigTxsError, msigTxsLoading, proposal])
 
-  const actionRequired = useMemo(() => {
-    if (proposalFoundError) return false
-    if (!props.walletAddress?.robust && !props.walletAddress?.id) return false
-    if (!!proposal && !stateLoading && !stateError) {
```

Comment:
All cases for returning false can be merged in one check, if `props.walletAddress` has no `id` or `robust`, `isAddressSigner` will already return false